images/clusterloader2/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.14.4 AS clusterloader-builder
+RUN go version
+WORKDIR /go/src/k8s.io
+RUN git clone https://github.com/njtran/perf-tests
+WORKDIR ./perf-tests/clusterloader2
+RUN mkdir bin
+RUN GOPROXY=direct go build -o ./bin/clusterloader ./cmd
```

Comment:
Nice find on GOPROXY, glad you figured that out. VPN is a pain. I'd probably just do go build ./cmd. Doesn't that provide a default output. I think it might be named "cmd" you'd have to check. On line 13, you can just copy that file.

Reply:
Hmm, I hadn't tried changing it. I'll keep it like this for now.

Reply:
Nevermind, misunderstood your point; changing.